runtime/queries/python/highlights.scm

```diff
@@ -111,6 +111,9 @@
 
 (integer) @constant.numeric.integer
 (float) @constant.numeric.float
+(expression_statement (string) @content
+	(#match? @content "^\"\"\"")
```

Comment:
@the-mikedavis wow, I guess we don't need grammar changes to detect this?

Reply:
It looks like python exposes a `(string_start)` node under `(string)` which contains these markers anyways, but it is handy that we wouldn't necessarily need the python grammar to do that in order to match this. (@probablySophie you can use `:tree-sitter-subtree` when selecting a syntax node to see how the nodes are laid out - this is usually more helpful than `:tree-sitter-scopes` which only shows the node names.)

I think we should follow nvim-treesitter's queries for this instead though since they're more specific and don't need `#match?` or `#eq?` at all: https://github.com/nvim-treesitter/nvim-treesitter/blob/dfd3207ed43a0c572c13acc02bf9d7dc270f4030/queries/python/highlights.scm#L201-L222